code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -9,7 +9,7 @@
 	var/life_tick = 0      // The amount of life ticks that have processed on this mob.
 
 	var/obj/item/restraints/handcuffs/handcuffed = null //Whether or not the mob is handcuffed, restraints/handcuffs/ required for var/cuffed_state
-	var/obj/item/legcuffed = null  //Same as handcuffs but for legs. Bear traps use this.
+	var/obj/item/restraints/legcuffs/legcuffed = null  //Same as handcuffs but for legs. Bear traps use this.
```

Comment:
Please convert to proper autodoc comment syntax while you're here

Reply:
Done!